### PR TITLE
Adding some details for how to install the connector

### PR DESCRIPTION
**Pull request by Paultagoras**
This isn't super long (though it should have all the steps) but it can serve as a solid start.

### CODE REVIEW

**Comment by robot-ch-test-poll** [comment]
This is an automated comment for commit 1759b1253fd20792f0871a389b0e1b8594674bdc with description of existing statuses. It's updated for the latest CI running
The full report is available [here](https://s3.amazonaws.com/clickhouse-test-reports/1246/1759b1253fd20792f0871a389b0e1b8594674bdc/ci_running.html)
The overall status of the commit is 🟢 success

<table><thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody><tr><td>Docs Check</td><td>Builds and tests the documentation</td><td>🟢 success</td></tr>
<tr><td>Vercel</td><td>There's no description for the check yet, please add it to tests/ci/ci_config.py:CHECK_DESCRIPTIONS</td><td>🟢 success</td></tr>
</table>

File: docs/en/integrations/data-ingestion/kafka/index.md [hunk 1]
```diff
@@ -823,6 +868,7 @@ ClickHouse Kafka Connect reports the following metrics:
 - Batch size is inherited from the Kafka Consumer properties.
 - When using KeeperMap for exactly-once and the offset is changed or rewound, you need to delete the content from KeeperMap for that specific topic.
```
Comment (mzitnik) [inline docs/en/integrations/data-ingestion/kafka/index.md:869]: Can you please provide how to do it or a link to somewhere in docs that this explained

**Review by mzitnik — APPROVED** [review]

File: docs/en/integrations/data-ingestion/kafka/index.md [hunk 2]
```diff
@@ -708,6 +708,51 @@ The [Kafka Connect](https://docs.confluent.io/platform/current/connect/index.htm
...
+```
+Example:
+<img src={require('./images/AddCustomConnectorPlugin.png').default} class="image" alt="Settings for adding a custom connector" style={{width: '50%'}}/>
+
+##### Configure the Connector
+Navigate to “Connectors” -> “Add Connector” and use the following settings (note that the values are examples only):
+
+```yml
+{
+  "database": "DATABASE_NAME",
```
Comment (mshustov) [inline docs/en/integrations/data-ingestion/kafka/index.md:738]: for consistency
```suggestion
  "database": "<DATABASE_NAME>",
```

File: docs/en/integrations/data-ingestion/kafka/index.md [hunk 3]
```diff
@@ -708,6 +708,51 @@ The [Kafka Connect](https://docs.confluent.io/platform/current/connect/index.htm
...
+  "topics": "<TOPIC_NAME>",
+  "username": "<SAMPLE_USERNAME>",
+  "key.converter": "org.apache.kafka.connect.storage.StringConverter",
+  "value.converter": "org.apache.kafka.connect.json.JsonConverter",
+  "value.converter.schemas.enable": "false"
+}
+```
+You should be all set!
+
+##### General Installation Instructions
```
Comment (mshustov) [inline docs/en/integrations/data-ingestion/kafka/index.md:755]: Could you add to known limitations that `consumer.*` settings cannot be overridden and we recommend contacting Confluent Support to adjust these?

File: docs/en/integrations/data-ingestion/kafka/index.md [hunk 4]
```diff
@@ -708,6 +708,51 @@ The [Kafka Connect](https://docs.confluent.io/platform/current/connect/index.htm
...
+  "port": "8443",
+  "ssl": "true",
+  "topics": "<TOPIC_NAME>",
+  "username": "<SAMPLE_USERNAME>",
+  "key.converter": "org.apache.kafka.connect.storage.StringConverter",
+  "value.converter": "org.apache.kafka.connect.json.JsonConverter",
+  "value.converter.schemas.enable": "false"
+}
+```
+You should be all set!
```
Comment (mshustov) [inline docs/en/integrations/data-ingestion/kafka/index.md:753]: I'd also add a troubleshooting section and call out that the error message in Confluent Logs UI doesn't provide enough details, so we recommend to download logs to investigate a problem. 
https://github.com/ClickHouse/clickhouse-kafka-connect/issues/124

File: docs/en/integrations/data-ingestion/kafka/index.md [hunk 5]
```diff
@@ -708,6 +708,51 @@ The [Kafka Connect](https://docs.confluent.io/platform/current/connect/index.htm
...
+#### Installing on Confluent Cloud (Bring-Your-Own-Connector)
+This is meant to be a quick guide to get you started with the ClickHouse Sink Connector on Confluent Cloud.
+For more details, please refer to the [official Confluent documentation](https://docs.confluent.io/cloud/current/connectors/bring-your-connector/custom-connector-qs.html#uploading-and-launching-the-connector).
+##### Create a Topic
+Creating a topic on Confluent Cloud is fairly simple, and there are detailed instructions [here](https://docs.confluent.io/cloud/current/client-apps/topics/manage.html).
+###### Important Notes
+* The Kafka topic name must be the same as the ClickHouse table name. The way to tweak this is by using a transformer (for example [ExtractTopic](https://docs.confluent.io/platform/current/connect/transforms/extracttopic.html)).
+* More partitions does not always mean more performance - see our upcoming guide for more details and performance tips.
+
+##### Add the Connector (BYOC)
```
Comment (mshustov) [inline docs/en/integrations/data-ingestion/kafka/index.md:720]:
```suggestion
##### Add Connector
```

File: docs/en/integrations/data-ingestion/kafka/index.md [hunk 6]
```diff
@@ -708,6 +708,51 @@ The [Kafka Connect](https://docs.confluent.io/platform/current/connect/index.htm
...
+'Connector type' - Sink
+'Sensitive properties' - 'password'. This will ensure entries of the ClickHouse password are masked during configuration.
+```
+Example:
+<img src={require('./images/AddCustomConnectorPlugin.png').default} class="image" alt="Settings for adding a custom connector" style={{width: '50%'}}/>
+
+##### Configure the Connector
+Navigate to “Connectors” -> “Add Connector” and use the following settings (note that the values are examples only):
+
+```yml
```
Comment (mshustov) [inline docs/en/integrations/data-ingestion/kafka/index.md:736]:
```suggestion
```json
```

**Review by mshustov — COMMENTED** [review]
We need to update https://clickhouse.com/docs/en/integrations/kafka#cloud-based-kafka-connectivity and https://clickhouse.com/docs/en/integrations/kafka#choosing-an-approach sections

**Review by mshustov — APPROVED** [review]
Let's add a warning how to access logs and we can merge the PR

File: docs/en/integrations/data-ingestion/kafka/index.md [hunk 7]
```diff
@@ -821,7 +870,8 @@ ClickHouse Kafka Connect reports the following metrics:
 ### Limitations
 - Deletes are not supported.
 - Batch size is inherited from the Kafka Consumer properties.
-- When using KeeperMap for exactly-once and the offset is changed or rewound, you need to delete the content from KeeperMap for that specific topic.
+- When using KeeperMap for exactly-once and the offset is changed or rewound, you need to delete the content from KeeperMap for that specific topic. (See troubleshooting guide below for more details)
+
 
 ### Troubleshooting
 #### "I would like to adjust the batch size for the sink connector"
```
Comment (DanRoscigno) [inline docs/en/integrations/data-ingestion/kafka/index.md:877]: I am guessing that this is still being worked on; please ping me when it is ready to be merged.

File: docs/en/integrations/data-ingestion/kafka/index.md (hunk 1 above)
Comment (rfraposa) [reply docs/en/integrations/data-ingestion/kafka/index.md]: Is this resolved yet?

**Comment by DanRoscigno** [comment]
sounds good to me

**Pull request merged**